### DISCUSSION TITLE
How to setup Mode 10 Course for Wheel Loader with Blade Leveller?

### DISCUSSION THREAD

**Discussion by cwattyeso** [Courseplay/courseplay#7438] [Q&A] [answered]
Okay so I've never really messed with the whole silage levelling and compacting aspect of Courseplay either in FS19 or previously in FS17. I've decided to give it a go on my latest map, to assist me with producing Compost easily in large volume. I have managed to play around with getting Unload Courses setup from my fields when doing straw and things and having a loading wagon empty those nicely for me in both lanes of my bunker, which is open both ends and allows for full drive through.

I've managed to play around and get Mode 9 working for a Wheel Loader with Bucket to empty my bunker with setting up a course for each lane to the compost storage point. So that's all good.

However mode 10 is giving me some headaches. At first I was trying it with a Tractor with Roller mounted on the back and couldn't understand why after recording my course, he would instantly reverse and start doing doughnuts in reverse away from the bunker. After much head scratching and finally reading the website tutorial I realised for Compacting with Roller I need to start with the roller in the bunker, tractor facing out and drive out and away from the bunker, put a wait point at a safe distance then reverse back and stop in front of the start point. Sure enough now the Tractor in Mode 10 will compact. However as more and more stuff gets delivered into the bunker eventually the tractor fails as the bunker needs really to be levelled.

I grabbed the Wheel Loader and got myself the Blade Leveller from the store. However I just can't figure out how to get courseplay to work with this. If I try starting with the Blade in the Bunker and reversing to a safe point, putting in a wait point and then switching back to forward and driving back to a point in front of the bunker. When I click start driving he doesn't do anything he just sits there. If I try recording a course with the wheel loader inside the bunker and driving out and away like with the tractor with roller, the same thing happens, when I hit start course he sits and does nothing. I've even tried variations where I drive the full length of the bunker and out the opposite end and do my wait point and reverse path, but again when I press the start button he just sits there.

Is it not possible to Use Mode 10 with this combination of vehicle and equipment? Does it only support rolling a bunker and not levelling?

**Comment by Tensuko** [comment, marked as answer]
Nah, a wheelloader won't do it.
Use this: https://farming-simulator.com/mod.php?lang=de&country=de&mod_id=124478&title=fs2019
You got to manually unfold it and start it, then start CP.

We can do 3 things with CP Mode10:
- Compacting: Any Vehicle, with or without a roller. We drive just up and down
- Leveling: With that Fliegl Duo, also basically just up/down.
- Pushing: With a blade to push everything in the back of the bunker.

We tried leveling with a blade, but that is not rly working, becouse you would have to adjust the blade hight rly rly fast to not dig holes, that is due to how FS handles the fruit on the ground.

**Comment by cwattyeso** [comment]
Ah okay thank you for that reply. So have just gone into game and sold the Levelling Blade and bought the Fliegl Duo for my tractor instead. Am I right in thinking I put the Fliegl on the front of the tractor now and leave the roller on the rear? I still use the same method for recording and creating the course? Meaning reverse into the bunker, start recording, drive out and away from the bunker, place a wait point then reverse back to the front of the bunker and stop recording? So effectively the Tractor is always driving through the bunker in reverse with the roller leading the way?

**Reply by nitroSkirata** [reply]
I think the duo needs to be on the rear and the roller on the front of the tractor, otherwise the Duo won't interact with the silage

**Reply by Tensuko** [reply]
Correct, the Duo works backwards, the rolls are just some weight as addition.

All compacting and stuff is done in reverse, so the tool that you want to use, should go in the back, unless you only put rolls, that it doesn't matter.
